common/views/components/EventPromo/EventPromo.js

```diff
@@ -147,6 +148,22 @@ const EventPromo = ({
           </Space>
         )}
       </CardBody>
+      <Divider extraClasses="divider--white divider--keyline" />
+      {event.secondaryLabels.length > 0 && (
+        <Space
+          v={{
+            size: 's',
+            properties: ['padding-top', 'padding-bottom'],
+          }}
+          h={{ size: 's', properties: ['padding-left', 'padding-right'] }}
+        >
+          <LabelsList
+            labels={event.secondaryLabels}
+            labelColor="black"
+            secondary={true}
```

Comment:
Could we just call this `roundedDiagonal` as the colour information isn't encapsulate in primary / secondary?